app/controllers/histories_controller.rb

```diff
@@ -16,7 +16,11 @@ def create
     # Increment number of contact attempts if applicable
     if params.permit(:type)[:type] == 'Contact Attempt'
       patient = current_user.patients.find(params.permit(:patient_id)[:patient_id])
-      patient&.update(contact_attempts: patient.contact_attempts + 1)
+      if params.permit(:comment)[:comment].downcase.include?('unsuccessful')
```

Comment:
So in theory, someone could log a "successful" contact attempt and use the word "unsuccessful" in the additional notes which would cause an issue here. It's unfortunate the history items themselves aren't tagged as successful/unsuccessful. I again think this points towards wanting Contact Attempts to be their own model rather than just a History item. This likely will make more sense in the future as well looking towards the History refactor we are planning on doing. I don't think this would be a big lift - but I don't want to take more of your time @timwongj since this isn't your task. @sgober Is that something you could maybe tackle if we move forward with it? Would just be adding a new model for ContactAttempts that has a successful bool and a comment field, and making sure the Patient model `has_many` of them and that they `belong_to` the Patient model, and then the History items can just pull that information from the ContactAttempt record. Thoughts?

Reply:
@kylietmo are you talking about accomplishing this sprint? because if so there's no way I will have time for it unfortunately. does it make sense to just make this it's own separate ticket and discuss the specs now so that it would be ready to go next sprint?